src/frontend/src/lib/stores/address.store.ts

```diff
@@ -1,10 +1,32 @@
 import { initCertifiedSetterStore } from '$lib/stores/certified-setter.store';
 import type { StorageStoreData } from '$lib/stores/storage.store';
-import type { Address } from '$lib/types/address';
+import type { Address, BtcAddress, EthAddress } from '$lib/types/address';
 import type { CertifiedData } from '$lib/types/store';
+import { writable, type Readable } from 'svelte/store';
 
-export type AddressData = CertifiedData<Address>;
+export type AddressData<T extends Address = Address> = CertifiedData<T>;
 
 export type OptionAddressData = StorageStoreData<AddressData>;
 
 export const addressStore = initCertifiedSetterStore<AddressData>();
+
+export type OptionCertifiedAddressData<T extends Address> = AddressData<T> | undefined | null;
```

Comment:
`OptionCertifiedAddressData` and `AddressStore` needs to be exposed (`export`)?

Reply:
i will un-expose in this PR, and check when I will need to expose them in the other one